src/models/msg/types.ts

```diff
@@ -1 +1 @@
-export type Categories = 'bank' | 'crisis' | 'distribution' | 'governance' | 'slashing' | 'staking' | 'profiles' | 'others';
+export type Categories = 'bank' | 'crisis' | 'distribution' | 'governance' | 'slashing' | 'staking' | 'profiles' | 'osmosis' | 'others';
```

Comment:
by module

Reply:
I have updated the code and added ```gamm``` ```incentives``` and ```lockup``` category